nnpdf_data/nnpdf_data/commondata/CMS_Z0_7TEV_DIMUON/metadata.yaml

```diff
@@ -31,13 +31,13 @@ implemented_observables:
     dataset_label: "CMS Drell-Yan 2D 7 TeV 2011"
```

Comment:
The only thing missing is that the `kinematics_override` needs to be set to `identity` in all of the files, because otherwise validphys will `sqrt` the masses (see e.g, the kinematic coverage plot, the scales are too low)

![image](https://github.com/user-attachments/assets/f05b7054-deb8-4449-b495-00e47d49e42f)